app/uk/gov/hmrc/alcoholdutyaccount/config/AppConfig.scala

```diff
@@ -27,14 +27,20 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig
 
   val appName: String = config.get[String]("appName")
 
-  val obligationDataHost: String            = servicesConfig.baseUrl("obligation")
+  private val obligationDataHost: String    = servicesConfig.baseUrl("obligation")
   val obligationDataToken: String           = getConfStringAndThrowIfNotFound("obligation.token")
   val obligationDataEnv: String             = getConfStringAndThrowIfNotFound("obligation.env")
+  private lazy val obligationDataUrlFormat  = new MessageFormat(
+    getConfStringAndThrowIfNotFound("obligation.url.obligationData")
+  )
```

Comment:
Since I fixed this for financial, I might as well complete by doing the url this way for obligations while I'm at it.